README.md

```diff
@@ -2,11 +2,21 @@
 
 ## Getting Started
 
+### 1. Install Dependencies
 ```js
 $ npm install
-$ npm start
 ```
 
+### 2. Add Environment file
+
+You'll need to add an `environment.json` file to set your path to the gecko repository.
+
+```json
+{ "geckoPath" : "/Users/jlaster/src/mozilla/gecko-dev"}
```

Comment:
Does this make sense to you?

I'm thinking other things like logging preferences, hot reloading could go here